Upstream change in GopherJS (Go 1.17->1.18) breaks pipeline

It seems that a recent update on `github.com/gopherjs/gopherjs` (Switch to Go 1.18: https://github.com/gopherjs/gopherjs/commit/f754ef8bf776018e09b416319246ed4b72365832) breaks the `GopherJS` build pipeline (seemingly because it uses Go 1.17), e.g. on this branch (PR https://github.com/otiai10/copy/pull/83):
https://github.com/otiai10/copy/runs/7760149468?check_suite_focus=true

```
# github.com/gopherjs/gopherjs/compiler
Error: compiler/package.go:403:14: fun.Type.TypeParams undefined (type *ast.FuncType has no field or method TypeParams)
Error: compiler/package.go:405:35: fun.Type.TypeParams undefined (type *ast.FuncType has no field or method TypeParams)
Error: compiler/package.go:492:55: named.TypeParams undefined (type *types.Named has no field or method TypeParams)
note: module requires Go 1.18
Error: Process completed with exit code 2.
```
I assume this is caused by the pipeline using master while at the same time specifying Go 1.17 for the deployment:
```
# This package only works fully with GopherJS as of 2c06401a, which is not yet
# released, thus we're pulling in `master`. Once GopherJS 1.18 is released with
# this support, we can depend on a proper tag.
name: GopherJS
...
  build:
    name: Build
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
	os: [ubuntu-latest]
        go: [1.17]
```

--- a/.github/workflows/gopherjs.yml
+++ b/.github/workflows/gopherjs.yml
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        go: [1.17]
+        go: [1.18]
     steps:
 
     - name: Set up Go
@@ -25,7 +25,7 @@ jobs:
       with:
         go-version: ${{ matrix.go }}
       id: go
-    
+
     - name: Install GopherJS
       run: |
         mkdir -p $(go env GOPATH)/src/github.com/gopherjs;

